src/system/catalog-sync.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Catalog Sync
+---
+
+Magento exports catalog data into tables using indexers. This process is automatically [triggered by several events]({% link system/index-management-events.md %}), such as changes to a product's price or inventory levels. Magento then syncs that catalog data to a central storage hub for use by various SaaS services, such as [Product Recommendations]({% link marketing/product-recommendations.md %}). The **Catalog Sync** dashboard allows you to peek into the Magento to SaaS syncing process and view the current status.
+
+With the **Catalog Sync** dashboard you can:
+
+-  View the sync status (**In Progress**, **Success**, **Failed**)
+-  View the total number of products synced, if successful
+-  Search and filter for products to view their update status
+-  Search across all store views by name, SKU, and so on
```

Comment:
you can only search one storeview at a time (you'll always be in the context of one specific storeview)